lib/generators/curly/scaffold/templates/model_presenter.rb.erb

```diff
@@ -0,0 +1,19 @@
+class <%= plural_table_name.capitalize %>::<%= singular_table_name.capitalize %>Presenter < Curly::Presenter
```

Comment:
You should probably be consistent and nest the class definition inside the module definition rather than using `::`.